text/0000-qa-team.md

```diff
@@ -0,0 +1,106 @@
+
+The day to day impact is more resources to empower contributors to make successful contributions to Grin by performing quality assurance testing and review on their own work. The long term impact is more consistency in the level of quality work that makes it into the Grin project over time.
+
+## Example
+
+In this example a large PR is submitted that makes a nontrivial change to the Grin node.
+* Before submitting the PR, the contributor:
+  * reads and integrates the well-formed contribution documentation to assist and support their efforts
+  * uses available testing frameworks to test their work
+* Reviewers of the PR may use the documentation and tooling support provided by the QA team to perform another round of quality assurance testing and review on the submission
+* Any necessary feedback is shared with the contributor to empower them to continue to make high quality contributions
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+## Membership
+
+* Membership is bootstrapped by @lehnberg and @j01tz
+* New membership approved with consensus of existing team members
+* Members may be removed by self-resignation, consensus decision by QA team members or by core team
+
+## Decision Making
+
+* A consensus-seeking approach is used for decision making through dialogue and discussion, modeled after the general principle in the Grin governance RFC[1].
+* In the event the team cannot reach consensus on a required action, the core team will determine the outcome
+
+## Responsibilities
+
+The responsibilities of the QA team _do_ include:
+  * Establishing and providing the necessary support for contributors to have a clear path to success by empowering them to perform quality assurance testing and review on their own work
+  * Clarifying, monitoring and enforcing the following of best QA practices by contributors
+
+The responsibilities of the QA team _do not_ include:
+  * Doing the quality assurance testing and review work themselves
+  * Writing tests for code submissions and changes
+  * Keeping testing frameworks updated based on codebase changes
+  * Manually reviewing each submission for quality assurance
+  * _Was the codebase changed? Then the tests must also be changed or extended for full coverage- this is not the responsibility of the QA team!_
+
+## Community Contributions
+
+The QA team is ultimately responsible for creating an environment that encourages and empowers contributors to make quality contributions to Grin. To this end, the QA team will strive to provide the resources necessary for contributors to make consistent, high quality contributions to Grin, including but not limited to: documentation, constructive feedback, tooling and clarity of expectations of quality.
```

Comment:
This seems a little bit at odds with what's stated above about what the responsibilities _do not_ include... for instance, surely the QA team will want to ensure their tooling and testing frameworks are up to date? Perhaps they won't be expected to maintain them 100% themselves, but the team would be the one striving to ensure they're in place, up-to-date and effective

Reply:
As it is currently written, my understanding is that the QA team is responsible for ensuring the tooling and frameworks are working before a contributor starts (this should be a function of the initial setup and ensuring that contributors don't make changes without adding working test coverage). 

The distinction is that the QA team is not responsible for updating the test framework to cover a change made by a contributor- the QA team is responsible for making sure that the contributor updates the test framework to cover their own change. This is supported by the QA team ensuring that expectations are documented, that the initial framework was set up properly and that changes from there were covered by their contributors.